src/components/dropdown/CustomRender.tsx

```diff
@@ -81,168 +69,127 @@ export const SingleValue = (props: any) => {
     props.selectProps?.parentInstance?.searchHeaderOption?.value;
   const rendererProps = { data: props.data };
   const isSearchHeaderSelected = props?.data?.searchHeader;
-  return (
-    <>
-      {isSearchHeaderSelected ? (
-        <components.SingleValue {...props}>
-          <div>{inputValue}</div>
-        </components.SingleValue>
-      ) : (
-        <>
-          {InputRenderer ? (
-            <components.SingleValue {...props}>
-              <InputRenderer {...rendererProps} />
-            </components.SingleValue>
-          ) : (
-            <components.SingleValue {...props} />
-          )}
-        </>
-      )}
-    </>
-  );
+  return (<>
+    {isSearchHeaderSelected ? (
```

Comment:
this could be refactored to set the child in a var

Reply:
done